frontend/src/utils.ts

```diff
@@ -23,3 +23,29 @@ export function deleteCookie(name: string): void {
   // Delete the cookie by setting the expiration date in the past
   document.cookie = `${name}=; expires=Thu, 01 Jan 1970 00:00:00 UTC; path=/;`;
 }
+
+function getMonthName(monthNumber: number) {
+  const monthNames = [
+    "Jan",
+    "Feb",
+    "Mar",
+    "Apr",
+    "May",
+    "Jun",
+    "Jul",
+    "Aug",
+    "Sep",
+    "Oct",
+    "Nov",
+    "Dec"
+  ];
+  return monthNames[monthNumber - 1];
+}
+
+export function formatedDate(x: string) {
+  const date = new Date(x);
+  const year = date.getFullYear();
+  const month = getMonthName(date.getMonth());
+  const day = date.getDate();
+  return `${month} ${day}, ${year}`;
+}
```

Comment:
- There exist a function named [`Intl.DateTimeFormat` ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat) for formatting dates in JS, no need to manually keep a list of months like this, please use that instead.
- Also the argument shoud be given a clear variable name (ex: `date`), not `x`
- Also, make this function accept a `string` or `Date` object as an argument.